### ISSUE TITLE
No version compatible with Raspberry Pi 

### ISSUE THREAD

**Issue by H0rze** [zacharee/ArcadyanKVD21Control#13]
There is no ARM64 version available to download. Are there plans to make a compatible version or am I missing anything?

**Comment by zacharee** [comment]
The packager I use doesn't support ARM64 Linux: https://github.com/hydraulic-software/conveyor/issues/8.

You might be able to use Box64 to run it: https://github.com/ptitSeb/box64.

**Comment by H0rze** [comment]
Would you still recommend downloading the .deb version or should I install another version?

**Comment by zacharee** [comment]
The tar.gz will probably be easier, since you'll be able to run the executable directly.

**Comment by H0rze** [comment]
I downloaded the tar.gz file and attempted to open arcadyankvd21control in the bin directory using box64, however, I got the following error message, ultimately stating it was aborted. I also tried it with Wine64 but it also didn't work. I am using a 64-bit os. Is there anything I am doing wrong or do I need to download anything?

Thanks for all the help so far!

![screenshot](https://github.com/zacharee/ArcadyanKVD21Control/assets/140589619/9ecc3ecc-6cad-4fef-b2c7-b1228068c20c)
